product-service/src/getAllProducts.ts

```diff
@@ -0,0 +1,14 @@
+import db from './db';
+import 'source-map-support/register';
+
+export const getAllProducts = async (_event, _context) => {
+    const products = await db.getProducts();
+    return {
+        statusCode: 200,
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+            'Access-Control-Allow-Credentials': true,
+        },
+        body: JSON.stringify(products)
```

Comment:
Duplicate of code. Please move to separate function

Reply:
Extracted to `addCorsHeaders` utilities and reused it in both functions.